modules/33acrossIdSystem.js

```diff
@@ -66,13 +74,17 @@ export const thirthyThreeAcrossIdSubmodule = {
    * decode the stored id value for passing to bid requests
    * @function
    * @param {string} id
-   * @returns {{'33acrossId':{ envelope: string}}}
+   * @returns {{'33acrossId':{ envelope: string}, <string>: { id: string, ext: Object }}}
```

Comment:
Are we denoting `<string>` to support other use cases for the ext response structure besides Magnite?

Reply:
yes.. basically leave the option to support more uses cases beside Magnite.. although we know that a new id won't be picked automatically, we need to add the corresponding entry to eids.js